test/API/H5_api_attribute_test.c

```diff
@@ -4301,7 +4301,7 @@ test_get_attribute_name(void)
         PART_BEGIN(H5Aget_name_by_idx_name_order_decreasing)
         {
             TESTING_2("H5Aget_name_by_idx by alphabetical order in decreasing order");
-#ifndef NO_DECREASING_ALPHA_ITER_ORDER
+            //#ifndef NO_DECREASING_ALPHA_ITER_ORDER
```

Comment:
@derobins We might have to consider a few other cases of capability flags since IIRC the DAOS VOL could only do alphabetical iteration on things in increasing order and not decreasing.